src/services/getChainInfo.ts

```diff
@@ -0,0 +1,46 @@
+import { SupportedChainId } from '../types';
+
+export interface BaseChainInfo {
```

Comment:
make it type. no need to have Base in the name

Reply:
Addressed [3256b73](https://github.com/Voltz-Protocol/v1-sdk/pull/211/commits/3256b731acf4597a7574c229a08393a743b505f9)